src/Kernel32/Kernel32+OSVERSIONINFO.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    /// <summary>
+    /// Contains the <see cref="OSVERSIONINFOEX"/> nested type.
+    /// </summary>
+    public static partial class Kernel32
+    {
+        /// <summary>
+        /// The <see cref="OSVERSIONINFO"/> structure contains operating system version information.
+        /// </summary>
+        /// <see href="https://docs.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-osversioninfow"/>
+        public unsafe struct OSVERSIONINFO
+        {
+            /// <summary>
+            /// The size of the <see cref="OSVERSIONINFO"/> structure in bytes.
+            /// </summary>
+            public int dwOSVersionInfoSize;
```

Comment:
I usually translate `DWORD `== [`unsigned long` (C++)](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/262627d8-3418-4627-9218-4ffe110850b2) => `uint `(C#).

Reply:
So do I, but then @AArnott mentioned that the guideline for this repository is to use `int` instead of `uint` if you can reasonably expect the value of the field to not exceed `int.MaxValue`.

Reply:
That makes sense!

Reply:
@qmfrederik is correct, but I usually add another condition: "...and it's likely that the value will be passed among other .NET APIs that accept `int`." If the value is rather peculiar (like a flags mask) that isn't likely to need to interop with other APIs, I usually stick with `uint` for the sake of matching the header file, just in case I was wrong in my estimate of how large the value might get.
FTR: I agree with `int` here.